react/src/components/atoms/buttons/ArrowButton/ArrowButton.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, text, select } from '@storybook/addon-knobs/react';
+import { action } from '@storybook/addon-actions';
+
+import ArrowButton from './index';
+import ArrowButtonDocs from './ArrowButton.md';
+
+storiesOf('atoms/buttons', module).addDecorator(withKnobs)
+  .add('Arrow Button', withInfo(`<div>${ArrowButtonDocs}</div>`)(() => {
```

Comment:
can you update this to `.add('ArrowButton', withInfo(`<div>${ArrowButtonDocs}</div>`)(() => {`

(no space)